### PR TITLE
fossGraphCollectionView table should be more informative

### PR DESCRIPTION
**Pull request by sanyavertolet**
### What's done:
 * Fixed table bottom margins
 * Replaced `userId` with `userInfo` in `VulnerabilityDto`
 * Added more info to `Vulnerability` table on `fossGraphCollectionView`
 * Fixed frontend permissions
 * Fixed placeholders on `fossGraph` view

### How it looks:
screenshot
screenshot
screenshot
screenshot

### CODE REVIEW

**Review by nulls — APPROVED** [review]

File: save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt [hunk 1]
```diff
@@ -94,57 +97,73 @@ class VulnerabilityService(
      * @param authentication [Authentication] describing an authenticated request
      * @return list of vulnerabilities with that match [filters]
      */
-    fun getFiltered(
+    @Transactional
```
[the remark this answers is not in the document]
Comment (sanyavertolet) [reply save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt:100]: makes sense

**Review by Cheshiriks — APPROVED** [review]

File: save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt [hunk 2]
```diff
@@ -94,57 +97,73 @@ class VulnerabilityService(
...
+            vulnerability.toDto().copy(userInfo = correspondingUserInfo ?: UserInfo("?"))
+        }
+    }
 
     /**
-     * @param filters filters for vulnerability
+     * @param filters
      * @return list of vulnerabilities with that match [filters]
      */
+    @Transactional(readOnly = true)
```
[the remark this answers is not in the document]
Comment (sanyavertolet) [reply save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt:149]: Removed

File: save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt [hunk 3]
```diff
@@ -94,57 +97,73 @@ class VulnerabilityService(
      * @param authentication [Authentication] describing an authenticated request
      * @return list of vulnerabilities with that match [filters]
      */
-    fun getFiltered(
+    @Transactional(readOnly = true)
```
[the remark this answers is not in the document]
Comment (Cheshiriks) [reply save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt]: No. You need to remove the entire annotation. You have now overridden the class annotation for this function and made it editable.

[the remark this answers is not in the document]
Comment (sanyavertolet) [reply save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt]: oh got it

[the remark this answers is not in the document]
Comment (sanyavertolet) [reply save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt]: now it is done!

**Pull request merged**